rest/adapter.go

```diff
@@ -78,7 +79,7 @@ func (v VariationMap) convert() *evaluation.VariationMap {
 }
 
 func convertTargetToIdentifier(tm []TargetMap) []string {
-	result := make([]string, 0)
+	result := make([]string, 0, len(tm))
 	for j := range tm {
 		result = append(result, *tm[j].Identifier)
```

Comment:
This is going to append to the end of result.
Instead you need to use the index, so the loop would look like this
```
for i, j := range tm {
    result[i] = j. Identifier
}
```